nhs/__main__.py

```diff
@@ -7,4 +7,4 @@
 from nhs.utils.logging import config_logger
 
 logger.enable("nhs")
-config_logger()
+config_logger()  # type: ignore
```

Comment:
what's the type error for `config_logger()`? I thought the function shouldn't raise any error

Reply:
Forgot to pass the configurations.yml file kms
![image](https://github.com/user-attachments/assets/81f72e0f-02b0-4dd8-9908-7f18034cb283)
Done